homeassistant/components/playstation_network/const.py

```diff
@@ -0,0 +1,8 @@
+"""Constants for the Playstation Network integration."""
+
+from datetime import timedelta
+
+DOMAIN = "playstation_network"
+DEVICE_SCAN_INTERVAL = timedelta(seconds=30)
```

Comment:
I think you can decrease this, afaik you can poll every 3 seconds (minimum allowed in core is 5s), so depends on how many times the coordinator polls per refresh interval...

Reply:
This is a limitation on the playstation side. Use of the api isn't officially supported by sony and the library strongly recommends a rate limit. If I'm recalling correctly it's 300 requests every 15 minutes. When a game is being played that means the integration makes about 180 requests in 15 minutes. There is some room to increase it, but I wanted to leave quite a bit of breathing room too.

Reply:
So you are doing 6 requests per coordinator refresh, right? Maybe this can be optimized a bit but we can leave that for later